src/Card/CardSection/CardSection.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Card from 'react-bootstrap/Card';
+
+const CardSection = ({
```

Comment:
Should we just rename Card Section to Card Body and let our card have multiple bodies?

Reply:
We could; this was actually a question I had during implementation as well since I replicating many of the styles Card Body already has into Card Section (i.e., padding).